e2e/sorting.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from "./common";
+
+/**
+ * - Creates a chart from the photovoltaik dataset
+ * - For each type of chart, changes the sorting between Name and Automatic
+ * - Checks that the legend item order is coherent.
+ */
+test("Segment sorting", async ({ selectors, actions, within, screen }) => {
+  test.setTimeout(60_000);
+
+  await actions.chart.createFrom(
+    "https://energy.ld.admin.ch/sfoe/bfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen/13",
+    "Int"
+  );
+
+  for (const chartType of ["Columns", "Lines", "Areas", "Pie"] as const) {
+    await actions.editor.changeChartType(chartType);
+    await actions.editor.selectActiveField("Color");
+
+    // Wait for color section to be ready
+    await selectors.edition.controlSection("Color").waitFor();
+
+    // Switch color on the first chart
+    if (chartType === "Columns") {
+      await within(selectors.edition.controlSection("Color"))
+        .getByText("None")
+        .click();
+
+      await actions.mui.selectOption("Standort der Anlage");
+    }
+
+    // Wait for chart to be loaded
+    await selectors.chart.loaded();
+    await selectors.edition.filtersLoaded();
+    await selectors.chart.colorLegend(undefined, { setTimeout: 5_000 });
+
+    const legendItems = await selectors.chart.colorLegendItems();
+    const legendTexts = await legendItems.allInnerTexts();
+    expect(legendTexts[0]).toEqual("Zurich");
+
+    if (chartType !== "Lines") {
```

Comment:
I think this condition also can be removed with the introduction of sorting to line chart?

Reply:
Done